### PR TITLE
huobi & bittrex - option name change (simple)

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

File: ts/src/pro/huobi.ts [hunk 1]
```diff
@@ -87,7 +87,9 @@ export default class huobi extends huobiRest {
                 'tradesLimit': 1000,
                 'OHLCVLimit': 1000,
                 'api': 'api', // or api-aws for clients hosted on AWS
-                'maxOrderBookSyncAttempts': 3,
```
Comment (carlosmiei) [inline ts/src/pro/huobi.ts:90]: @ttodua one thing is the orderbook fetching, and another is the orderbook synching, not sure if we should use a single option for both or not

Comment (ttodua) [reply ts/src/pro/huobi.ts:90]: yes, that's I've mentioned too, so to ask - what should we name it? `snapshotSyncMaxRetries`? while we have `snapshotMaxRetries` for solely "fetching" retries.
or we should rename again. 
lmk your thoughts

Comment (ttodua) [reply ts/src/pro/huobi.ts:90]: also, for existing `snapshotMaxRetries`, imo, can be renamed into `snapshotFetchMaxRetries`.

Comment (kroitor) [reply ts/src/pro/huobi.ts:90]: @carlosmiei @ttodua I may be missing something, but if we're going to unify this option, then why not just call it `maxRetries` or something else as simple as that? When it comes to orderbook subscriptions there's just one process that "can be tried", that is synching the orderbook.

```
'options': {
    'watchOrderBook': {
        'maxRetries': 3,
    },
}
```

Comment (carlosmiei) [reply ts/src/pro/huobi.ts:90]: @kroitor 
>  there's just one process that "can be tried", that is synching the orderbook.

Actually there are two, fetching the orderbook using rest (that can fail due to a temporary issue) and second align that orderbook with the deltas we cached

Comment (carlosmiei) [reply ts/src/pro/huobi.ts:90]: Of course we can merge both options and make it one

Comment (ttodua) [reply ts/src/pro/huobi.ts:90]: @carlosmiei i've merged, you can take a look now.

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**